src/data/AdminForm.js

```diff
@@ -8,7 +8,7 @@ export const affiliations = [
   "marketing",
 ];
 
-export const requirementsData = {
+export const requirement = {
```

Comment:
```suggestion
export const prerequisites = {
```

Reply:
could you update the import and then change the requirements, setRequirements to prerequisites, setPrerequisites